src/main.rs

```diff
@@ -173,6 +191,22 @@ fn main() {
             .expect("'start-time-cpu_us' parameter expected to be of 'u64' type.")
     });
 
+    if let Some(logs_path) = cmd_arguments.value_of("logs-path") {
+        let logger_json = format!(
+            r#"{{
+                "log_fifo": "{}",
+                "metrics_fifo": "{}"
+            }}"#,
+            create_path(logs_path),
```

Comment:
I missing to understand why are you creating the path to the log/metrics file/fifo for the user? When the logger is configured through the API, we are expecting the user to have the paths created beforehand.

Reply:
I am creating the path only if it doesn't exist already to give the user more a little bit more flexibility.

Reply:
I understand, however this would mean providing different behaviors when configuring through API vs configuration through this newly defined parameters. My proposal is to keep the configuration of the logger consistent on all levels. @firecracker-microvm/compute-capsule needing your input here.